app/views/components/general-page-description.njk

```diff
@@ -0,0 +1,12 @@
+{% macro generalPageDescription(titleText, descriptionText) %}
```

Comment:
Is there a unit test for this new component?

Reply:
yeah I missed it off by mistake - should be there now 👍